docs/UserGuide.md

```diff
@@ -587,72 +629,82 @@ Format: `clear`
 
 ## FAQ
 
-**Q**: How do I start using Productiv?<br>
-**A**: You can refer to our [Quick Start Guide](#quick-start).
+**Q**: Is Productiv available on both Windows and Mac?<br>
+**A**: Absolutely! Just ensure that you have Java `11` installed on your computer and it is your default Java version.
+
+**Q**: How many deliverables, meetings and contacts can I store in Productiv?<br>
+**A**: Productiv can support up to 1000 deliverables, 1000 meetings and 1000 contacts.
+
+**Q**: Can I transfer my data to another device?<br>
```

Comment:
I honestly feel that this FAQ seems abit too complicated to explain to users and is kinda prone to errors and messy instructions.
1. what does "device runs Productiv" mean? do we need to refer to quick start guide again?
2. is it 'copy over' or 'overwrite' ? and would they understand what these mean and the difference?
3. is it "./data" or just "data"

I personally prefer to take out this FAQ to minimise the communication breakdown. 4 FAQ left should be fine (they are explained well and concisely too). What do you think?

Reply:
yep, this particular FAQ needs some step by step instructions, which shouldn't be the point of the FAQ.
I feel can remove.